main/main.c

```diff
@@ -1737,7 +1737,7 @@ int php_request_startup(void)
 		if (PG(max_input_time) == -1) {
 			zend_set_timeout(EG(timeout_seconds), 1);
```

Comment:
actually, this is also an input timeout: when `max_input_time` equals to `-1ű, the value of `max_execution_time` is used (https://www.php.net/manual/en/info.configuration.php#ini.max-input-time)